bentoml/service_env.py

```diff
@@ -152,20 +152,22 @@ def set_setup_sh(self, setup_sh_path_or_content):
         else:
             self._setup_sh = setup_sh_path_or_content.encode("utf-8")
 
-    def set_pip_dependencies(self, pip_dependencies):
+    def add_pip_dependencies(self, pip_dependencies):
         if not isinstance(pip_dependencies, list):
             pip_dependencies = [pip_dependencies]
         self._pip_dependencies += pip_dependencies
 
-    def set_requirements_txt(self, requirements_txt_path_or_content):
-        requirements_txt_file = Path(requirements_txt_path_or_content)
+    def set_requirements_txt(self, requirements_txt_path):
+        requirements_txt_file = Path(requirements_txt_path)
 
         if requirements_txt_file.is_file():
             with requirements_txt_file.open("rb") as f:
-                self._requirements_txt = f.read()
+                content = f.read()
+                module_list = content.decode('utf-8').split('\n')
+                self._pip_dependencies += module_list
         else:
-            self._requirements_txt = requirements_txt_path_or_content.encode(
-                "utf-8"
+            raise BentoMLException(
```

Comment:
Use ValueError

Reply:
Actually - just remove the if statement in front, and let it raise FileNotFoundError